docs/Documentation/openvino_legacy_features.md

```diff
@@ -0,0 +1,95 @@
+    advanced operations on models, such as Model conversion API, Benchmark Tool, 
+    Accuracy Checker, Annotation Converter, Post-Training Optimization Tool, 
+    and Open Model Zoo tools. Most of these tools have been either removed, 
+    replaced by other solutions, or moved to the OpenVINO Runtime package.
+|   :doc:`See how to install Development Tools <openvino_docs_install_guides_install_dev_tools>`
+
+
+| **Model Optimizer**
+|   *New solution:* Direct model support and OVC
+|   *Old solution:* Model Optimizer discontinuation planned for 2024
+
+|   Model Optimizer's role was largely reduced when all major model frameworks became
+    supported directly. For the sole purpose of converting model files explicitly, 
+    it has been replaced with a more light-weight and efficient solution, the
+    OpenVINO Converter (launched with OpenVINO 2023.1).
+
+.. :doc:`See how to use OVC <?????????>`
+
+
+| **Open Model ZOO**
+|   *New solution:* users are encouraged to use public model repositories
+|   *Old solution:* discontinuation planned for OpenVINO 2024.0
+|
+|   Open Model ZOO provided a collection of models prepared for use with OpenVINO,
+    and a small set of tools enabling a level of automation for the process.
+    Since the tools have been mostly replaced by other solutions and several
+    other model repositories have recently grown in size and popularity,
+    Open Model ZOO will no longer be maintained. You may still use its resources
+    until they are fully removed.  
+|   :doc:`See the Open Model ZOO documentation <model_zoo>`
+|   `Check the OMZ GitHub project <https://github.com/openvinotoolkit/open_model_zoo>`__
+
+
+| **MX Net, Caffe, and Kaldi model formats**
+|   *New solution:* conversion to ONNX will be the only option
+|   *Old solution:* conversion of these models to OpenVINO IR will be unsupported  
+
+|   Since these three model formats proved to be far less popular among OpenVINO users 
+    than the remaining ones, their support has been discontinued. Converting them to the
+    ONNX format is a possible way of retaining them in the OpenVINO-based pipeline.
+|   :doc:`See the previous conversion instructions <mxnet_caffe_kaldi>`
+|   :doc:`See the currently supported frameworks <Supported_Model_Formats>`
+
+
+
+| **Post-training Optimization Tool (POT)**
+|   *New solution:* NNCF extended in OpenVINO 2023.0
+|   *Old solution:* POT discontinuation planned for 2024
+|    
+|   Neural Network Compression Framework (NNCF) now offers the same functionality as POT,
+    apart from its original feature set. It is currently the default tool for performing 
+    both, post-training and quantization optimizations, while POT is considered deprecated.
+ 
+|   :doc:`See how to use NNCF for model optimization <openvino_docs_model_optimization_guide>`
+|   `Check the NNCF GitHub project, including documentation <https://github.com/openvinotoolkit/nncf>`__
+
+| **new API 2.0**
```

Comment:
this header doesn't follow the name pattern, where the header is the old solution, kind of gives impression that we're removing new api